docs/pages/component/props.mdx

```diff
@@ -788,6 +791,15 @@ source={{
   }}
 ```
 
+### `showSubtitleButton`
```

Comment:
it would be pretty easier to to move this prop inside: (controlsStyles)[https://thewidlarzgroup.github.io/react-native-video/component/props#controlsstyles]
Can you please move it ?
Can I suggest another behavior:
showSubtitleButton -> default true 
If true display the subtitle button only when subtitle track are available
If false never display the subtitle button
(it is maybe already the case, but would be great to clarify it in the doc)

Reply:
I couldn't agree more. Let me do it.